__tests__/app.test.js

```diff
@@ -0,0 +1,52 @@
+const request = require("supertest")
+const app = require("../db/app")
+const db = require("../db/connection")
+const seed = require("../db/seeds/seed")
+const data = require("../db/data/test-data")
+
+beforeEach(() => {
+    return seed(data)
+})
+
+afterAll(() => {
+    return db.end()
+})
+
+describe('GET/API', () => {
+    test('200 OK: message', () => {
+        return request(app)
+        .get('/api')
+        .expect(200)
+        .then(({body}) => {
+            expect(body.msg).toBe("All OK")
+        })
+    })
+})
+
+describe('GET/API/TOPICS', () => {
+    test('200 OK: all topics', () => {
+        return request(app)
+        .get('/api/topics')
+        .expect(200)
+        .then(({body}) => {
+            const { topics } = body
+             expect(topics).toBeInstanceOf(Object)
+             expect(topics).toHaveLength(3)
+             topics.forEach((topic) => {
+                 expect(topic).toMatchObject({
+                     description: expect.any(String),
+                     slug: expect.any(String)
+                 })
+             })
+        })
+    })
+})
+
+describe('GET /api/topics (ERROR)', () => {
```

Comment:
I would change this description - you are right to have this test in it's own describe block but it is not just linked to /api/topics, it is basically any endpoint that doesn't match. The outcome should be the same whether it's /api/invalid or /nonsense. Maybe rename this to route not found